odk1-src/briefcase-using.rst

```diff
@@ -175,15 +175,47 @@ Export forms to CSV
 
     - :guilabel:`Export media files` enables exporting media files into the chosen export directory
     - :guilabel:`Overwrite existing files` enables overwriting form instance data in the output files. The default behavior is to append data.
-    - :guilabel:`Split select multiples` enables splitting select multiple fields. Enabling this setting will create an extra output column per select choice, with a `1` if the choice was selected, or `0` otherwise. This only affects select fields without a choice filter and that are not from an external file.
+    - :guilabel:`Split select multiples` enables splitting select multiple fields. Enabling this setting will create an extra output column per select choice, with a `1` if the choice was selected, or `0` otherwise. This only affects select fields without a choice filter and that are not from an external file (including widgets with `search appearance`_).
+    - :guilabel:`Include GeoJSON export` enables generating a companion GeoJSON output export file with spatial data from all exported submissions.
     - :guilabel:`Pull before export` enables trying to pull the selected forms in case there are new form instances to be exported.
 
+    .. _search appearance: https://odk-demo.readthedocs.io/en/latest/form-widgets.html#select-with-search-widget
```

Comment:
Not sure why this is there...

Reply:
Huh... I didn't realize about that. I haven't been able to find that in the current docs... could it be that we don't document that search appearance anymore?